.github/scripts/build_triton_wheel.py

```diff
@@ -91,7 +94,7 @@ def build_triton(
         patch_init_py(
             triton_pythondir / "triton" / "__init__.py",
             version=f"{version}",
-            expected_version=None,
+            expected_version=read_triton_version(device),
```

Comment:
The method `read_triton_version` is used in many place in this file, and we can not ensure only this place needs the 'device' argument in future. So I suggest to set a global var `TRIOTN_VERSION_FILE` and set it according to the device in the `main` method before `build_triton`.